common/functions-v2/tests/it/scalars/string.rs

```diff
@@ -42,6 +43,18 @@ fn test_upper(file: &mut impl Write) {
     )]);
 }
 
+fn test_lower(file: &mut impl Write) {
+    run_ast(file, "lower('Abc')", &[]);
+    run_ast(file, "lower('DOBRÝ DEN')", &[]);
+    run_ast(file, "lower('ß😀山')", &[]);
```

Comment:
Let's try something fancy:

```suggestion
    run_ast(file, "lower('İ😀山')", &[]);
```